web_src/js/components/ActivityHeatmap.vue

```diff
@@ -7,7 +7,6 @@
             {{ totalContributions }} total contributions in the last 12 months
         </h4>
         <calendar-heatmap v-show="!isLoading" :locale="locale" :no-data-text="locale.no_contributions" :tooltip-unit="locale.contributions" :end-date="endDate" :values="values" :range-color="colorRange"/>
-        <div class="ui divider"></div>
```

Comment:
Doesn't this remove the divider on the home page?

Reply:
![image](https://user-images.githubusercontent.com/13721712/82320744-0f7a2100-99d4-11ea-853a-40a73424f266.png)

^ The one between the first activity message and the heatmap

Reply:
True, too bad, it does, but removes the double-divider look on the activity page, which is worse IMO. This is probably more of a hotfix...

Reply:
Hmm but it should not break the changes intentionally introduced by  #11437 (which it does if you're just reverting it)